### ISSUE TITLE
Munging for JAVA 9

### ISSUE THREAD

**Issue by 0b1kn00b** [HaxeFoundation/haxe#10581]
``` 
as of release 9, '_' is a keyword, and may not be used as an identifier
```

**Comment by Simn** [comment]
That's annoying...

**Comment by Simn** [comment]
The Java target will be put in "regression fixes only" state with the Haxe 4.3.0 release. See https://github.com/HaxeFoundation/haxe/wiki/Target-Support-Level#java-and-c-since-430 for additional information. We apologize for any inconvenience this causes!
